### PR TITLE
Add default configs section with default host

### PR DESCRIPTION
**Pull request by eylonronen**
Solves #649

### CODE REVIEW

File: cmd/lakectl/cmd/root.go [hunk 1]
```diff
@@ -131,5 +131,8 @@ func initConfig() {
 	viper.SetEnvKeyReplacer(strings.NewReplacer(".", "_")) // support nested config
 	viper.AutomaticEnv()                                   // read in environment variables that match
 
+	//Configuration defaults
```
Comment (johnnyaug) [inline cmd/lakectl/cmd/root.go:134]:
```suggestion
	// Configuration defaults
```

File: cmd/lakectl/cmd/root.go [hunk 2]
```diff
@@ -131,5 +131,8 @@ func initConfig() {
...
+	viper.SetDefault("server.endpoint_url", "http://127.0.0.1:8000")
```
Comment (johnnyaug) [inline cmd/lakectl/cmd/root.go:135]: Note that server.endpoint_url is already defined as a constant at the top of this file.
I think the value itself could also go as a separate constant, "DefaultServerEndpointURL".

**Review by johnnyaug — CHANGES_REQUESTED** [review]
Hi @eylonronen!
Welcome and thank you for your effort and this PR.
See minor change request

**Review by guy-har — COMMENTED** [review]
looks good 👍

**Pull request closed without merging**